### PR TITLE
Apply php/doc-en@63b9908

### PR DESCRIPTION
**Pull request by saundefined**

### CODE REVIEW

File: language-snippets.ent [hunk 1]
```diff
@@ -2143,6 +2143,11 @@ Selon le comportement souhaité, l'argument doit être transtypé en &string; ou
  <function>mysqli_real_escape_string</function>.</para>
 </warning>'>
 
+<!ENTITY mysqli.conditionalexception '<para xmlns="http://docbook.org/ns/docbook">
+Si le signalement des erreurs mysqli est activé (<constant>MYSQLI_REPORT_ERROR</constant>) et que l&#39;opération demandée échoue,
```
Comment (devnexen) [inline language-snippets.ent:2147]: `Si le rapport d'erreurs mysqli`

File: language-snippets.ent [hunk 2]
```diff
@@ -2683,6 +2688,33 @@ Par défaut, le paramètre <parameter>notext</parameter> vaut &true;.
 
 <!ENTITY reflection.export.param.name 'La réflection à exporter.'>
 
+<!ENTITY reflection.getattributes.param.name '<varlistentry xmlns="http://docbook.org/ns/docbook">
+<term><parameter>name</parameter></term>
+<listitem>
+ <para>
+  Filtrez les résultats pour inclure uniquement les instances
```
Comment (devnexen) [inline language-snippets.ent:2695]: Filtrer

File: language-snippets.ent [hunk 3]
```diff
@@ -1166,7 +1166,7 @@ Chaque champs est converti en type PHP approprié, sauf :
  <entry>8.1.0</entry>
  <entry>
   La paramètre <parameter>imap</parameter> attend désormais une instance de
-  <classname>IMAP\Connection</classname> ; auparavant, une &resource; était attendu.
+  <classname>IMAP\Connection</classname> ; auparavant, une <literal>imap</literal> &resource; était attendu.
```
Comment (devnexen) [inline language-snippets.ent:1169]: `attendu` => `attendue`, same remarks below.

File: language-snippets.ent [hunk 4]
```diff
@@ -1031,7 +1031,7 @@ Ils doivent être considérés <emphasis>obsolète</emphasis>, et devraient seul
  <entry>8.0.0</entry>
  <entry>
   <parameter>image</parameter> attend une instance de <classname>GdImage</classname> désormais;
-  auparavant, une <type>resource</type> était attendu.
+  auparavant, une <literal>gd</literal> <type>resource</type> était attendu.
```
Comment (devnexen) [inline language-snippets.ent:1034]: `une gd resource` => `une ressource gd`, similar-ish comments for below

File: language-snippets.ent [hunk 5]
```diff
@@ -1260,23 +1260,23 @@ constante <literal>IntlChar::PROPERTY_*</literal>).</para>'>
  <entry>8.1.0</entry>
  <entry>
   Le paramètre <parameter>ldap</parameter> attend désormais une instance de
-  <classname>LDAP\Connection</classname> ; auparavant, une &resource; était attendu.
+  <classname>LDAP\Connection</classname> ; auparavant, une &resource; <literal>ldap link</literal> était attendu.
```
Comment (devnexen) [inline language-snippets.ent:1263]: there is still `attendu` occurrences 🙂

File: language-snippets.ent [hunk 6]
```diff
@@ -1031,7 +1031,7 @@ Ils doivent être considérés <emphasis>obsolète</emphasis>, et devraient seul
  <entry>8.0.0</entry>
  <entry>
   <parameter>image</parameter> attend une instance de <classname>GdImage</classname> désormais;
-  auparavant, une <type>resource</type> était attendu.
+  auparavant, une <type>resource</type> <literal>gd</literal> était attendu.
```
Comment (devnexen) [inline language-snippets.ent:1034]: here

File: language-snippets.ent [hunk 7]
```diff
@@ -4058,7 +4090,7 @@ xmlns="http://docbook.org/ns/docbook"><simpara>Cette fonction a été
  <entry>8.0.0</entry>
  <entry>
   <parameter>parser</parameter> attend une instance de <classname>XMLParser</classname>
-  désormais; auparavent, une <type>resource</type> était attendu.
+  désormais; auparavent, une <type>resource</type> <literal>xml</literal> était attendu.
```
Comment (devnexen) [inline language-snippets.ent:4093]: and here.

File: language-snippets.ent (hunk 5 above)
Comment (saundefined) [reply language-snippets.ent]: Thanks! :)

**Review by adoy — APPROVED** [review]

**Review by Girgias — APPROVED** [review]
LGTM, just to double check the MongoDB changes for language-snippets are after ``63b99082ef83eade08151f8cb528246fded81db9``?

**Pull request merged**